code/game/machinery/suit_storage_unit.dm

```diff
@@ -60,6 +60,10 @@
 	storage_type = /obj/item/weapon/watertank/atmos
 
 /obj/machinery/suit_storage_unit/mining
+	suit_type = /obj/item/clothing/suit/hooded/explorer
+	mask_type = /obj/item/clothing/mask/gas
+
+/obj/machinery/suit_storage_unit/mining/eva

```

Comment:
What? You can also select its Z5 in the same file.

So yeah 3 lines, but all in one file that is never touched.